test/autoprefixer.test.js

```diff
@@ -547,6 +547,13 @@ it('works with CSS Modules', () => {
   postcss([autoprefixer()]).process(':export { selectors: _1q6ho_2 }').css
 })
 
+it('ignore prefix in vendor at rules', () => {
+  let input = read('at-rules')
```

Comment:
We have a shortcut to do this test. You can find it in `hacks` secltion in this test file.

Reply:
I fixed that.